### PR TITLE
[testing] Lib injection and System-tests integration

### PR DESCRIPTION
**Pull request by robertomonteromiguel**
### What does this PR do?
We are moving lib injection tests to system-tests repository

### Motivation
unified system-tests 

### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

**Review by rochdev — COMMENTED** [review]
Change looks fine, but the tests are failing.

**Comment by robertomonteromiguel** [comment]
> 
Yes, the problem was in system-tests...now it is fixed

File: .github/workflows/test-k8s-lib-injection.yaml [hunk 1]
```diff
@@ -8,69 +8,67 @@ on:
...
+      - name: Checkout nodejs tracer
+        uses: actions/checkout@v3
+        with:
+          path: './dd-trace-js'
+ 
+      - name: Checkout system tests
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+        with:
+          repository: 'DataDog/system-tests'
+          ref: 'robertomonteromiguel/lib_injection_integration'
```
Comment (robertomonteromiguel) [inline .github/workflows/test-k8s-lib-injection.yaml:40]: I will remove this:
* When my system-tests branch is merged to main branch
* Just before merging this PR

File: .github/workflows/test-k8s-lib-injection.yaml [hunk 2]
```diff
@@ -8,69 +8,67 @@ on:
     - cron: '0 4 * * *'
 
 jobs:
-  lib-injection-test:
+  injection-tests-nodejs:
```
Comment (robertomonteromiguel) [inline .github/workflows/test-k8s-lib-injection.yaml:11]: I decided not to remove this file...but another option it is moving these jobs to system-tests.yml
what do you think?

**Review by rochdev — APPROVED** [review]

**Review by Qard — APPROVED** [review]

File: .github/workflows/test-k8s-lib-injection.yaml [hunk 3]
```diff
@@ -21,56 +22,86 @@ jobs:
...
-          init-image-version: ${GITHUB_SHA}
+            init-image-version: ${GITHUB_SHA}
 
-      - name: Build sample app image
+      - name: Push snapshot image
         run: |
-          docker build k8s/sample-app --build-arg npm_pkg=./dd-trace.tgz -t ghcr.io/datadog/dd-trace-js/sample-app:${GITHUB_SHA}
-          docker push ghcr.io/datadog/dd-trace-js/sample-app:${GITHUB_SHA}
+          docker buildx build --platform=linux/amd64,linux/arm/v7,linux/arm64/v8 -t ghcr.io/datadog/dd-trace-js/dd-lib-js-init:latest_snapshot --push lib-injection
+        if: ${{ steps.set_names.outputs.image_name }} == 'robertomonteromiguel-lib_injection_system_tests_integration'
```
Comment (robertomonteromiguel) [inline .github/workflows/test-k8s-lib-injection.yaml:41]: I will change to master before merge

**Review by Qard — APPROVED** [review]

**Pull request merged**